Microsoft.Toolkit.Uwp.SampleApp/SamplePages/AdaptiveGridView/definition.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Name":  "AdaptiveGridView",
+    "Type":  "AdaptiveGridViewPage",
+    "About":  "Presents items in a evenly-spaced set of columns to fill the total available display space. It reacts to changes in the layout as well as the content so it can adapt to different form factors automatically.",
+    "CodeUrl":  "https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView",
+    "XamlCodeFile":  "AdaptiveGridViewCode.bind",
+    "Icon":  "/SamplePages/AdaptiveGridView/AdaptiveGridView.png",
```

Comment:
Can we just use the name of the image and force images to be part of the sample folder?